docker/frontend/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM node:18-alpine AS base
+
+# Install dependencies only when needed
+FROM base AS deps
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+WORKDIR /app
+
+# Install dependencies
+RUN echo "$PWD"
```

Comment:
This shouldn't be needed for the build, but possibly produces an extra layer. If it was intended for debugging, I would suggest to remove it.